### ISSUE TITLE
Clarification about memory benchmarks.

### ISSUE THREAD

**Issue by AndreasPK** [haskell-perf/graphs#8]
I think "space" and "data size" could use one more sentence to explain what they actually benchmark.

For example I would have expected space to measure maximum memory usage not allocations. Only after lookup up what weigh does it became clearer that it's about allocations.

Similar "data size" was not at all self explanatory to me (until looking up the package).

Otherwise great work and I will use this as another data point to benchmark my GHC changes!

**Comment by nobrakal** [comment]
> I think "space" and "data size" could use one more sentence to explain what they actually benchmark.

Indeed. It will require again some work, because latest criterion comes with some tools to inspect the memory usage/allocation (see http://hackage.haskell.org/package/criterion-1.5.1.0/docs/Criterion-Types.html#t:Measured ). I prefer to solve this before changing documentation ^^.

Concerning `ghc-heap-size`, it fails to build wtih recent GHC and a part of it has been included. I will have anyway to refactor this part of code !

> Otherwise great work and I will use this as another data point to benchmark my GHC changes!

Thanks, it is great to see that this repo can be useful !
